Core/HLE/sceNet.cpp

```diff
@@ -59,11 +235,30 @@ static void __ResetInitNetLib() {
 
 void __NetInit() {
 	portOffset = g_Config.iPortOffset;
+	if (PPSSPP_ID == 0) // For persistent ID/IP, New instances may have a possibility to have the same ID/IP if PPSSPPIDCleanup() being called on every reset
+	{
+		PPSSPP_ID = getPPSSPPInstanceNumber(); // This should be called when program started instead of when the game started/reseted
+	}
+#ifdef _WIN32
+	WSADATA data;
+	int iResult = WSAStartup(MAKEWORD(2, 2), &data);
+	if (iResult != NOERROR) {
+		ERROR_LOG(SCENET, "WSA Failed");
+	}
+#endif
+	InitLocalIP();
+	INFO_LOG(SCENET, "LocalHost IP will be %s", inet_ntoa(((sockaddr_in*)&localIP)->sin_addr));
+	//net::Init();
 	__ResetInitNetLib();
 }
 
 void __NetShutdown() {
 	__ResetInitNetLib();
+	//net::Shutdown();
+#ifdef _WIN32
+	WSACleanup();
```

Comment:
I don't think we want this, or the WSAStartup?  We already start networking on Windows, so in theory this is just doing nothing.  At worst, it might make other parts of PPSSPP that use networking crash, if something gets out of sync.  Doesn't seem like there's a good reason to have it.

-[Unknown]

Reply:
Yes it should be safe to remove, actually it's no longer there on my final code.

PS: The only issue i had with WSAStartup/Shutdown was with UPnP fallback cleanup (within PortManager class destructor), but it should never reach that fallback if UPnP is being cleanup properly

Reply:
Yeah I see it was removed in "Added UPnP support". I will do the same here, so hopefully the conflict will be none or very easy once you rebase.